<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWindow</class>
 <widget class="QMainWindow" name="SetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pushButtonSearch:hover {
background-color: rgb(153, 153, 153);
border-style: inset;
}

QPushButton#pushButtonConnect:hover {
border-style: inset;
}

QPushButton#pushButtonDisconnect:hover {
border-style: inset;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semilight</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButtonSearchr{
background-color: #999999;
color: rgb(50, 50, 50);
}
#pushButtonSearch:pressed{
background-color: #4d4d4d; 
}</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxDevices">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Yu Gothic UI Semilight&quot;;
color: rgb(50, 50, 50);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Yu Gothic UI Semilight&quot;;</string>
      </property>
      <property name="title">
       <string>Logs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonConnect">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semilight</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButtonConnect {
background-color:#00ff00;
color: rgb(50, 50, 50);
font: 11pt &quot;Yu Gothic UI Semilight&quot;;
}

#pushButtonConnect:pressed{
background-color:#00b300;
}</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonDisconnect">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semilight</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButtonDisconnect {
background-color: rgb(255, 26, 26);
color: rgb(50, 50, 50);
font: 11pt &quot;Yu Gothic UI Semilight&quot;;
}
#pushButtonDisconnect:pressed {
background-color: #b30000;
}

</string>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>297</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
